adafruit_ltr390.py

```diff
@@ -316,11 +330,11 @@ def _reset(self):
             i2c.write(bytes((_THRESH_LOW, 0x00, 0x00, 0x00)))
 
     @property
-    def _mode(self):
+    def _mode(self) -> Literal[0, 1]:
```

Comment:
`Literal[0, 1]` evaluates to `bool` so we should use that in almost all circumstances (unless something specific, I may have recommended it in the past but I think in retrospect it was a bad recommendation, sorry!).  This makes sense since the return represents a the state of a bit from a hardware level.